src/Illuminate/Database/Schema/Grammars/SQLiteGrammar.php

```diff
@@ -36,7 +37,22 @@ class SQLiteGrammar extends Grammar
      */
     public function compileTableExists()
     {
-        return "select * from sqlite_master where type = 'table' and name = ?";
+        return "select sql from sqlite_master where type = 'table' and name = ?";
+    }
+
+    /**
+     * Compile the query to determine the SQL text that describes the given object.
+     *
+     * @param  string  $name
+     * @param  string  $type
+     * @return string
+     */
+    public function compileSqlCreateStatement($name, $type = 'table')
+    {
+        return sprintf('select "sql" from sqlite_master where type = %s and name = %s',
```

Comment:
```suggestion
        return sprintf('select sql from sqlite_master where type = %s and name = %s',
```

12 lines above, you use it also without quote-marks.